rcljava/src/main/java/org/ros2/rcljava/events/Event.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.ros2.rcljava.events;
+
+import java.lang.ref.WeakReference;
+
+import org.ros2.rcljava.interfaces.Disposable;
+import org.ros2.rcljava.events.EventStatus;
+
+/**
+ * This class serves as a bridge between ROS2's rcl_event_t and RCLJava.
```

Comment:
nit: I think you can drop "ROS2" since rcljava is also ROS 2.
```suggestion
 * This class serves as a bridge between a rcl_event_t and RCLJava.
```

I wonder if we should name the class `RCLEvent` to make it clear this pertains to `rcl`-level events?

Reply:
> I wonder if we should name the class RCLEvent to make it clear this pertains to rcl-level events?

This is actually not an event but an event handler, so I was planning to rename this `EventHandler`.

It's true that event is a generic term that could be used to describe several different things, but I find repeating `RCL` annoying.
`rclcpp` uses the name `QOSEventHandler`.
I don't like much that name, is [MessageLost](https://github.com/ros2/rclcpp/blob/6a3a5ed841562b6e371f3cf8cc6d9201cf070c80/rclcpp/include/rclcpp/qos_event.hpp#L45) a qos event?

IDK, I'm a bit confused about naming in this case ....
If `EventHandler` sounds too generic, I think we could go the rclcpp way and rename it to `QoSEventHandler`.

Reply:
I think `EventHandler` is good. I agree that prepending `RCL` everywhere is a bit redundant; afterall, this is **rcl**java.

Reply:
Renamed in https://github.com/osrf/ros2_java/pull/4/commits/51c640f03f3fc9b2fd968589c3e09f7bd4515295.